lib/repl.js

```diff
@@ -223,6 +229,32 @@ module.paths = CJSModule._nodeModulePaths(module.filename);
 const writer = (obj) => inspect(obj, writer.options);
 writer.options = { ...inspect.defaultOptions, showProxy: true };
 
+// Converts static import statement to dynamic import statement
+const toDynamicImport = (codeLine) => {
+  let dynamicImportStatement = '';
+  let moduleName = '';
+  const ast = acornParse(codeLine, { __proto__: null, sourceType: 'module', ecmaVersion: 'latest' });
+  const toCamelCase = (str) => SideEffectFreeRegExpPrototypeSymbolReplace(/[-_](\w)/g, str, (_, c) => StringPrototypeToUpperCase(c));
```

Comment:
This should not be necessary, we should not modify what the user has passed, can you please remove it?
```suggestion
```